### PR TITLE
Improve generated inline docs

### PR DESCRIPTION
**Pull request by andrewvc**
Improved docs for Elasticsearch output v2.0. This changeset fixes grammatical errors and uses more clear language than that which preceded it.

### CODE REVIEW

File: lib/logstash/outputs/elasticsearch.rb [hunk 1]
```diff
@@ -10,44 +10,32 @@
...
-# always make it successfully. For example, there could be events which are not formatted
-# correctly for the index they are targeting (type mismatch in mapping). So that we minimize loss of 
-# events, we have a specific retry policy in place. We retry all events which fail to be reached by 
-# Elasticsearch for network related issues. We retry specific events which exhibit errors under a separate 
-# policy described below. Events of this nature are ones which experience ES error codes described as 
-# retryable errors.
-#
-# *Retryable Errors:*
+# This plugin uses the Elasticsearch bulk API to optimize its imports into Elasticsearch. These requests may experience
+# either partial or total failures. Events are retried if they fail due to either a network error, or the status codes
```
Comment (jordansissel) [inline lib/logstash/outputs/elasticsearch.rb]: no comma after 'error,'

File: lib/logstash/outputs/elasticsearch.rb [hunk 2]
```diff
@@ -10,44 +10,32 @@
...
 #
-# This output only speaks the HTTP, which is the preferred protocol for interacting with Elasticsearch. By default
-# Elasticsearch exposes HTTP on port 9200.
-#
-# We strongly encourage the use of HTTP over the node protocol. It is just as
-# fast and far easier to administer. For those wishing to use the java protocol please see the 'elasticsearch_java' gem.
+# This output only speaks the HTTP protocol. HTTP is the preferred protocol for interacting with Elasticsearch as of Logstash 2.0.
+# We strongly encourage the use of HTTP over the node protocol for a number of reasons. HTTP is only marginally slower,
+# yet far easier to administer and work with. When using the HTTP protocol one may upgrade Elasticsearch versions without having
+# to upgrade Logstash in lock-step. For those wishing to use the java protocol please see the 'elasticsearch_java' gem.
```
Comment (jordansissel) [inline lib/logstash/outputs/elasticsearch.rb]: Call it "node protocol" here maybe? This is the first mention of "java protocol" I think.

**Comment by jordansissel** [comment]
Two tiny comments. Otherwise love it. :)

**Comment by andrewvc** [comment]
@jordansissel Fixed! (thanks for the feedback). Do I have an LGTM?

**Comment by jordansissel** [comment]
LGTM

File: lib/logstash/outputs/elasticsearch.rb [hunk 3]
```diff
@@ -361,16 +349,23 @@ def submit(actions)
...
+          status = props['status']
+          error = props['error']
+
+          if RETRYABLE_CODES.include?(status)
+            @logger.warn "retrying failed action with response code: #{status}"
             actions_to_retry << action
-          elsif not SUCCESS_CODES.include?(resp_code)
-            @logger.warn "failed action with response of #{resp_code}, dropping action: #{action}"
+          elsif not SUCCESS_CODES.include?(status)
+            @logger.warn "failed action", status: status, error: error, action: action
```
Comment (jordansissel) [inline lib/logstash/outputs/elasticsearch.rb]: we should use hash rocket, and () for calls.

**Comment by jordansissel** [comment]
(minor comment left, LGTM still stands, fix if you want, still looking good!)

**Pull request merged**